tasks/eth/3-add-superchainConfig/Validation.md

```diff
@@ -60,6 +77,53 @@ There should also be a single 'State Override' in the Foundation Safe contract
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** The GnosisSafe `approvedHashes` mapping is updated to indicate approval of this transaction by the council. The correctness of this slot can be verified as follows:
+    - Since this is a nested mapping, we need to use `cast index` twice to confirm that this is the correct slot. The inputs needed are:
+      - The location (`8`) of the `approvedHashes` mapping in the [GnosisSafe storage layout](https://github.com/safe-global/safe-contracts/blob/v1.4.0/contracts/libraries/SafeStorage.sol#L23)
+      - The address of the Council Safe: `0xc2819DC788505Aac350142A7A707BF9D03E3Bd03`
+      - The safe hash to approve: `0x782cc13743013d9b1b0854b914bba5ebde49971466bf982c98d4ba911eb0d42d`
+    - The using `cast index`, we can verify that:
+      ```shell
+        $ cast index address 0xc2819DC788505Aac350142A7A707BF9D03E3Bd03 8
+        0xaaf2b641eaf0bae063c4f2e5670f905e1fb7334436b902d1d880b05bd6228fbd
+        ```
+        and
+      ```shell
+        $ cast index bytes32 0x782cc13743013d9b1b0854b914bba5ebde49971466bf982c98d4ba911eb0d42d 0xaaf2b641eaf0bae063c4f2e5670f905e1fb7334436b902d1d88
+        0xc17968c40bf9fa0af0c9c957e0f95fb7d057c959f119d672b696ef249c039704
+        ```
+      And so the output of the second command matches the key above.
+
+#### For the Foundation:
+
+- **Key:** `0x66dfee7d20e8ae2c45828a6a3c2c79c377eccb8b4cea869195a802469fe70584` <br/>
+  **After:** `0x0000000000000000000000000000000000000000000000000000000000000001` <br/>
+  **Meaning:** The GnosisSafe `approvedHashes` mapping is updated to indicate approval of this transaction by the council. The correctness of this slot can be verified as follows:
+    - Since this is a nested mapping, we need to use `cast index` twice to confirm that this is the correct slot. The inputs needed are:
+      - The location (`8`) of the `approvedHashes` mapping in the [GnosisSafe storage layout](https://github.com/safe-global/safe-contracts/blob/v1.4.0/contracts/libraries/SafeStorage.sol#L23)
+      - The address of the Council Safe: `0x5a0aae59d09fccbddb6c6cceb07b7279367c3d2a`
+      - The safe hash to approve: `0x782cc13743013d9b1b0854b914bba5ebde49971466bf982c98d4ba911eb0d42d`
+    - The using `cast index`, we can verify that:
+      ```shell
+        $ cast index address 0x847B5c174615B1B7fDF770882256e2D3E95b9D92 8
+        0x13908ba1c0e379ab58c6445554ab471f3d4efb06e3c4cf966c4f5e918eca67bd
+      ```
+      and
+      ```shell
+        $ cast index bytes32 0x782cc13743013d9b1b0854b914bba5ebde49971466bf982c98d4ba911eb0d42d 0x13908ba1c0e379ab58c6445554ab471f3d4efb06e3c4cf966c4f5e918eca67bd
+        0x66dfee7d20e8ae2c45828a6a3c2c79c377eccb8b4cea869195a802469fe70584
+      ```
+      And so the output of the second command matches the key above.
 
 ### `0x5a7749f83b81b301cab5f48eb8516b986daef23d` (`L1ERC721BridgeProxy`)
```

Comment:
_(arbitrary comment location to start a thread)_ 

I did not validate existing state changes, as presumably thr prior reviewer validated those

Reply:
I think that's valid.